Client/Frontend/Home/Wallpapers/v1/Interface/WallpaperManager.swift

```diff
@@ -119,7 +123,7 @@ class WallpaperManager: WallpaperManagerInterface, FeatureFlaggable, Loggable {
                 completion(.success(()))
             } catch {
                 browserLog.error("Error fetching wallpaper resources: \(error.localizedDescription)")
-                completion(.failure(error))
+                completion(.failure(WallpaperManagerError.downloadFailed(error)))
```

Comment:
are these errors worth logging in sentry?

Reply:
I don't think so.